identity-iota/src/tangle/client_map.rs

```diff
@@ -102,7 +113,7 @@ impl Default for ClientMap {
   }
 }
 
-#[async_trait::async_trait(?Send)]
+#[async_trait::async_trait(? Send)]
```

Comment:
nit: IDE formatting thing.

```suggestion
#[async_trait::async_trait(?Send)]
```